.drone.yml

```diff
@@ -36,6 +36,27 @@ steps:
     when:
       event:
         - pull_request
+        
+  - name: fossa
+    image: registry.suse.com/suse/sle15:15.3
+    failure: ignore
+    environment:
+      FOSSA_API_KEY:
+        from_secret: FOSSA_API_KEY
+    commands:
+      - zypper -n install curl unzip
+      - "curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/spectrometer/master/install.sh | sh"
```

Comment:
Should we be always using the master install script? Or should we be pinning this to help minimize chance of failure?